### PR TITLE
Intersection Point in Y Shapped Linked Lists

### PR DESCRIPTION
**Pull request by Rishikeshrajrxl**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

( In this program I have taken value from user for two Linked List. Then I have intersect it manually from user's choice.
   At last I have find the Point of intersection in Y shaped Linked List.)

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#495 )

### CODE REVIEW

**Comment by Rishikeshrajrxl** [comment]
@plazzy99 
@siddharth25pandey 
@iamrajiv  
I have completed this assignment, please review this and let me know if any changes needed.

File: C-Plus-Plus/ds/Intersected Linked List.cpp [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+@created: 20/08/2020
+*/
+
+#include <iostream>
+using namespace std;
+struct node
+{                                //declaration of node
+int data;
+struct node*next1,*next2;
+}*head1=NULL,*head2=NULL,*temp1,*temp2,*ptr;
```
Comment (vatsalkesarwani12) [inline C-Plus-Plus/ds/Intersected Linked List.cpp]: make use of proper spacing for better readability of code

File: C-Plus-Plus/ds/Intersected Linked List.cpp [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+           default:
+           cout<<"Invalid Input"<<endl;
+       };
+   }while(1);
+   return 0;
+}                                //  End of Main() function here 
+//-----------------------------------------------------------------------------
+void Insert_1()
+{
+    ptr=(struct node*)malloc(sizeof(struct node));
```
Comment (vatsalkesarwani12) [inline C-Plus-Plus/ds/Intersected Linked List.cpp]: when you are using cpp why using **malloc** use **new**

File: C-Plus-Plus/ds/Intersected Linked List.cpp [hunk 3]
```diff
@@ -0,0 +1,219 @@
+/*
+Intersection Point in Y Shapped Linked Lists.
+@author: Rishikeshrajrxl
+@created: 20/08/2020
+*/
+
```
Comment (vatsalkesarwani12) [inline C-Plus-Plus/ds/Intersected Linked List.cpp:7]: import cmath header to let **abs** function to work

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
edit readme file

**Comment by vatsalkesarwani12** [comment]
Best input/output message format seen till date🤩

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Rishikeshrajrxl  Instead of importing any header file.
just write <bits/stdc++.h>

**Comment by Rishikeshrajrxl** [comment]
> edit readme file

I have made other pull request regarding update of ReadMe.me file 
#630. please follow this changes.

**Comment by Rishikeshrajrxl** [comment]
@plazzy99  , Done with all changes, please review this.

**Comment by Rishikeshrajrxl** [comment]
I have also updated  CONTRIBUTING.md file for the issue #542 .
In this I have written documentation  with proper screenshot that can actually help new contributors to understand how they can start contributing to this Project.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1 +1,57 @@
-# Contributing to NeoAlgo
+# Contributing Guidlines
+### Learn how to create issues, Fork a repo, makes changes, and ask the maintainers to review and merge it
+## Creating Pull Request
+You have a GitHub repo and can push to it. All is well. But how the heck do you contribute to other people's GitHub projects? That is what I wanted to know after I learned git and GitHGub. I will explain how to fork a git repo, make changes, and submit a pull request.
```
Comment (atarax665) [inline CONTRIBUTING.md]: Use Grammarly to fix all the typos and grammatical errors.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1 +1,57 @@
...
+
+- Click New issue. 
+  <img src="img/new_issue.png" /><br>
+- If there are multiple issue types, click Get started next to the type of issue you'd like to open.
+  <img src="img/issue_template_get_started_button.png" /><br>
+  - Type a title and description for your issue.
+  - After creating the issue you have to wait until the project maintainer assign the issue to you.
+  - when you're finished, click Submit new issue.
+  <br>
+
```
Comment (atarax665) [inline CONTRIBUTING.md]: Add all the rules mentioned in #398 for Contributor' Hack participants.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by Rishikeshrajrxl** [comment]
Done all changes, please review.

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
why so much file?

**Comment by iamrajiv** [comment]
@Rishikeshrajrxl please remove unnecessary files and resolve merge conflicts

**Pull request closed without merging**